app/views/clients/coordinated_entry_assessments/edit.haml

```diff
@@ -1,13 +1,11 @@
-- title = "Editing #{_'Coordinated Entry Assessment'} for #{@client.pii_provider(user: current_user).full_name}"
+- title = "Editing #{_'Coordinated Entry Assessment'}"
```

Comment:
These look like some translations we missed.  I _think_ this should be 
```suggestion
- title = "Editing #{Translation.translate('Coordinated Entry Assessment')}"
```

Reply:
thanks! - sounds like there's more to unpack around how we use translations, seems like an area for future documentation.